arcgis-ios-sdk-samples/Scenes/Surface placements/SurfacePlacementsViewController.swift

```diff
@@ -20,62 +20,116 @@ import ArcGIS
 /// A view controller that manages the interface of the Surface Placements
 /// sample.
 class SurfacePlacementsViewController: UIViewController {
-    /// The scene displayed in the scene view.
-    let scene: AGSScene
+    // MARK: Instance properties
     
-    required init?(coder: NSCoder) {
-        scene = AGSScene(basemap: .topographic())
-        
-        // add base surface for elevation data
+    /// A label to show the value of the slider.
+    @IBOutlet weak var zValueLabel: UILabel!
+    /// The slider to change z-value of `AGSPoint` geometries, from 0 to 140 in meters.
+    @IBOutlet weak var zValueSlider: UISlider!
+    /// The segmented control to toggle the visibility of two draped mode graphics overlays.
+    @IBOutlet weak var drapedModeSegmentedControl: UISegmentedControl!
+    
+    /// The scene view managed by the view controller.
+    @IBOutlet var sceneView: AGSSceneView! {
+        didSet {
+            sceneView.scene = makeScene()
+            sceneView.setViewpointCamera(AGSCamera(latitude: 48.3889, longitude: -4.4595, altitude: 80, heading: 330, pitch: 97, roll: 0))
+            // Add graphics overlays of different surface placement modes to the scene.
+            let surfacePlacements: [AGSSurfacePlacement] = [
+                .drapedBillboarded,
+                .drapedFlat,
+                .relative,
+                .relativeToScene,
+                .absolute
+            ]
+            let overlays = surfacePlacements.map(makeGraphicsOverlay)
+            overlaysBySurfacePlacement = Dictionary(uniqueKeysWithValues: zip(surfacePlacements, overlays))
+            sceneView.graphicsOverlays.addObjects(from: overlays)
+        }
+    }
+    
+    /// A dictionary for graphics overlays of different surface placement modes.
+    var overlaysBySurfacePlacement = [AGSSurfacePlacement: AGSGraphicsOverlay]()
+    /// A formatter to format z-value strings.
+    let zValueFormatter: MeasurementFormatter = {
+        let formatter = MeasurementFormatter()
+        formatter.unitStyle = .short
+        formatter.unitOptions = .providedUnit
```

Comment:
In other samples I've not specified this option so that the users sees the measurement in the unit that makes the most sense for their locale. It is up to you, but I would recommend doing that.

Reply:
Hmm I tend to leave it as is. If not specified, system locale would use miles as unit, which is not very readable (number always read around 0.1 miles). 😳 
There are a few other samples with it specified, such as Viewshed, LineOfSight, Animate 3D, etc.

Reply:
There is [`naturalScale`](https://developer.apple.com/documentation/foundation/measurementformatter/unitoptions/1642065-naturalscale), which would fix the problem of the measurement not being very readable. With that option, I'm pretty sure you'd get measurements in feet.

Reply:
Nice to earn a new thing everyday 😄